src/index.html

```diff
@@ -4,9 +4,519 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <div class="header__container">
+        <a href="/" class="header__logo">
+          <img
+            src="images/logo.png"
+            alt="logo"
+          >
+        </a>
+        <nav class="nav">
```

Comment:
```suggestion
        </a>
        
        <nav class="nav">
```
leaving blank lines between multiline siblings is a good idea though. check everywhere